### ISSUE TITLE
Unlock  character size

### ISSUE THREAD

**Issue by pan-mroku** [Julow/Unexpected-Keyboard#531]
Hello,

I'd like to be able to set more than 1.5x in "character size" setting. How can I unlock it? Modifying a config file manually on the phone could also be a solution

**Comment by Spike-from-NH** [comment]
Concur that the maximum should let the letter fill the keycap. However, you may regret this if you go on to define swipes in your custom layout; there would be no room for their legends.

**Comment by pan-mroku** [comment]
Yes, using both the character size I'd want and swipes, would be unwieldy. However in my use case there will be no swipe layout. The characters would be too small. And from what I've looked around, it seems that Unexpected is the only keyboard that has the option of custom layouts.

And before someone suggests that route: I know that android has some accessibility options but I've looked at them and in my case these won't work.

If only I could set that setting to 2x-3x (I'm not sure how will it scale further) instead of 1.5x, that would solve my problem

**Comment by Spike-from-NH** [comment]
That's interesting. But you've said forever that you want appearance to be defined separately from layout.

**Comment by Julow** [comment]
That's true. But some symbols need to be made smaller or dimmer (for example, the Esc key or the bottom row. I'm not suggesting to specify the full appearance of a symbol this way.

**Comment by Spike-from-NH** [comment]
If you're going to break your rule, go for a clean break!

All you needed to do to satisfy me, you already did—letting me go to 1.30× legends. But my dream is still `STYLE="css"` in the custom layouts. Breaks your rule, yes, but it seems that's not still a problem?

**Comment by Julow** [comment]
I'm not suggesting to break the rule.
I'm talking about flags that controls some aspects of the symbols, not a new styling system. Layouts and themes should stay separate.

**Comment by pan-mroku** [comment]
> Layouts and themes should stay separate

Is there a way to customise theme more than the settings?

**Comment by Julow** [comment]
@pan-mroku Not yet. It has been requested several times in the past so any contribution in that direction might be popular.

**Comment by Spike-from-NH** [comment]
> Layouts and themes should stay separate

This is an elegant principle. But a different elegance, from my point of view, is: "All user customization (layout or appearance) should be able to reside in a single file on my device."
